src/TLuaInterpreter.cpp

```diff
@@ -11035,6 +11047,581 @@ int TLuaInterpreter::restartIrc(lua_State* L)
     return 1;
 }
 
+#ifdef QT_TEXTTOSPEECH_LIB
+
+/** ttsSpeak( text )
+ *  Synthesizes text for reading.
+ */
+int TLuaInterpreter::ttsSpeak(lua_State* L)
+{
+    TLuaInterpreter::ttsBuild();
+    
+    if (!lua_isstring(L, 1)) {
+        lua_pushfstring(L, "ttsSpeak: argument expected string, got %s!", luaL_typename(L, 1));
```

Comment:
This isn't consistent with our error messages which explain the wrong argument location and what it should be - I'll fix this example, could you cover the rest @jgh713 ?

Reply:
This seems resolved now, right? @vadi2

Reply:
Yes